package_linux.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# Packages the Python app with all its dependencies for Linux x86-64 platform.
```

Comment:
_"Packages the Python app"_ -> _"Packages the app"_ or _"Packages the application"_? The same is relevant for `package_macos.sh`, `package_windows.sh`.

Reply:
Done.